src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorPickerSlider.cs

```diff
@@ -0,0 +1,286 @@
+						 args.Key != VirtualKey.Left &&
+						 args.Key != VirtualKey.Right) ||
+				(this.Orientation == Orientation.Vertical &&
+						 args.Key != VirtualKey.Up &&
+						 args.Key != VirtualKey.Down))
+			{
+				base.OnKeyDown(args);
+				return;
+			}
+
+			ColorPicker parentColorPicker = GetParentColorPicker();
+
+			if (parentColorPicker == null)
+			{
+				return;
+			}
+
+			bool isControlDown = (Window.Current.CoreWindow.GetKeyState(VirtualKey.Control) & CoreVirtualKeyStates.Down) == CoreVirtualKeyStates.Down;
+
+			double minBound = 0;
+			double maxBound = 0;
+
+			Hsv currentHsv = parentColorPicker.GetCurrentHsv();
+			double currentAlpha = 0;
+
+			switch (this.ColorChannel)
+			{
+				case ColorPickerHsvChannel.Hue:
+					minBound = parentColorPicker.MinHue;
+					maxBound = parentColorPicker.MaxHue;
+					currentHsv.H = this.Value;
+					break;
+
+				case ColorPickerHsvChannel.Saturation:
+					minBound = parentColorPicker.MinSaturation;
+					maxBound = parentColorPicker.MaxSaturation;
+					currentHsv.S = this.Value / 100;
+					break;
+
+				case ColorPickerHsvChannel.Value:
+					minBound = parentColorPicker.MinValue;
+					maxBound = parentColorPicker.MaxValue;
+					currentHsv.V = this.Value / 100;
+					break;
+
+				case ColorPickerHsvChannel.Alpha:
+					minBound = 0;
+					maxBound = 100;
+					currentAlpha = this.Value / 100;
+					break;
+
+				default:
+					throw new Exception("Invalid ColorPickerHsvChannel."); // Uno Doc: 'throw winrt::hresult_error(E_FAIL);'
+			}
+
+			bool shouldInvertHorizontalDirection = this.FlowDirection == FlowDirection.RightToLeft && !this.IsDirectionReversed;
+
+			ColorHelpers.IncrementDirection direction =
+				((args.Key == VirtualKey.Left && !shouldInvertHorizontalDirection) ||
+				 (args.Key == VirtualKey.Right && shouldInvertHorizontalDirection) ||
+				  args.Key == VirtualKey.Up) ?
+				ColorHelpers.IncrementDirection.Lower :
+				ColorHelpers.IncrementDirection.Higher;
+
+			ColorHelpers.IncrementAmount amount = isControlDown ? ColorHelpers.IncrementAmount.Large : ColorHelpers.IncrementAmount.Small;
+
+			if (this.ColorChannel != ColorPickerHsvChannel.Alpha)
+			{
+				currentHsv = ColorHelpers.IncrementColorChannel(currentHsv, this.ColorChannel, direction, amount, false /* shouldWrap */, minBound, maxBound);
+			}
+			else
+			{
+				currentAlpha = ColorHelpers.IncrementAlphaChannel(currentAlpha, direction, amount, false /* shouldWrap */, minBound, maxBound);
+			}
+
+			switch (this.ColorChannel)
+			{
+				case ColorPickerHsvChannel.Hue:
+					this.Value = currentHsv.H;
+					break;
+
+				case ColorPickerHsvChannel.Saturation:
+					this.Value = currentHsv.S * 100;
+					break;
+
+				case ColorPickerHsvChannel.Value:
+					this.Value = currentHsv.V * 100;
+					break;
+
+				case ColorPickerHsvChannel.Alpha:
+					this.Value = currentAlpha * 100;
+					break;
+
+				default:
+					throw new Exception("Invalid ColorPickerHsvChannel."); // Uno Doc: 'MUX_ASSERT(false);'
```

Comment:
Codacy found an issue: ['System.Exception' should not be thrown by user code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6121259)